src/compass/s1_geo_stack.py

```diff
@@ -0,0 +1,510 @@
+        Low sub-band bandwidth for split-spectrum in Hz
+    high_band: float
+        High sub-band bandwidth for split-spectrum in Hz
+    '''
+    error = journal.error('s1_geo_stack_processor.main')
+    info = journal.info('s1_geo_stack_processor.main')
+
+    # Check if SLC dir and DEM exists
+    if not os.path.isdir(slc_dir):
+        err_str = f'{slc_dir} SLC directory does not exist'
+        error.log(err_str)
+        raise FileNotFoundError(err_str)
+
+    if not os.path.isfile(dem_file):
+        err_str = f'{dem_file} DEM file does not exists'
+        error.log(err_str)
+        raise FileNotFoundError(err_str)
+
+    # Create directory for runfiles
+    run_dir = f'{work_dir}/run_files'
+    os.makedirs(run_dir, exist_ok=True)
+
+    # Check if orbit are provided, if Not download
+    if orbit_dir is None:
+        info.log('Orbit directory not assigned. Download orbit ephemerides')
+
+        # Create orbit dir and check internet connection
+        orbit_dir = f'{work_dir}/orbits'
+        os.makedirs(orbit_dir, exist_ok=True)
+        check_internet_connection()
+
+        # List all zip file and extract info
+        zip_file_list = sorted(glob.glob(f'{slc_dir}/S1*zip'))
+
+        for zip_file in zip_file_list:
+            sensor_id, _, start_datetime, \
+            end_datetime, _ = parse_safe_filename(zip_file)
+
+            # Find precise orbits first
+            orbit_dict = get_orbit_dict(sensor_id, start_datetime,
+                                        end_datetime, 'AUX_POEORB')
+            # If orbit_dict is empty, precise orbits have not been found. Find restituted orbits instead
+            if orbit_dict == None:
+                orbit_dict = get_orbit_dict(sensor_id, start_datetime,
+                                            end_datetime, 'AUX_RESORB')
+            # Download the orbit file
+            download_orbit(orbit_dir, orbit_dict['orbit_url'])
+
+    # Find reference date and construct dict for geocoding
+    if ref_date is None:
+        ref_file = sorted(glob.glob(f'{slc_dir}/S1*zip'))[0]
+    else:
+        ref_file = glob.glob(f'{slc_dir}/S1*{str(ref_date)}*zip')[0]
+
+    # Generate burst map and prune it if a list of burst ID is provided
+    burst_map = generate_burst_map(ref_file, orbit_dir, x_spac, y_spac, epsg)
+
+    if burst_id is not None:
+        burst_map = prune_dataframe(burst_map, 'burst_id', burst_id)
+
+    # Start to geocode bursts
+    zip_file = sorted(glob.glob(f'{slc_dir}/S1*zip'))
+    for safe in zip_file:
+        i_subswath = [1, 2, 3]
+        orbit_path = get_orbit_file_from_list(safe,
+                                              glob.glob(f'{orbit_dir}/S1*'))
+
+        for subswath in i_subswath:
+            bursts = load_bursts(safe, orbit_path, subswath)
+            for burst in bursts:
+                if burst.burst_id in list(set(burst_map['burst_id'])):
+                    runconfig_path = create_runconfig(burst, safe, orbit_path,
```

Comment:
Currently runconfig_path is relative path. It would be great to make it absolute path